app/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml.scala

```diff
@@ -46,14 +45,25 @@ object DeclarationXml {
         <GoodsItemQuantity>{dec.declarationType.totalNumberOfItems}</GoodsItemQuantity>
         <TotalPackageQuantity>55</TotalPackageQuantity>
         <AdditionalDocument>
```

Comment:
can we instead do something like
`{ dec.documentation.additionalPayment.map(maybeAdditionalPayment) }` ?

Reply:
This doesn't allow for situation where user has entered a space.